### PR TITLE
Make counsel-yank-pop ivy-height configurable

### PR DESCRIPTION
**Pull request by jojojames**
From https://github.com/abo-abo/swiper/issues/1362

### CODE REVIEW

File: counsel.el [hunk 1]
```diff
@@ -3133,6 +3133,11 @@ A is the left hand side, B the right hand side."
   :group 'ivy
   :type 'string)
 
+(defcustom counsel-yank-pop-ivy-height 5
```
Comment (basil-conto) [inline counsel.el]: Maybe `counsel-yank-pop-height` would be clear enough?

File: counsel.el [hunk 2]
```diff
@@ -3133,6 +3133,11 @@ A is the left hand side, B the right hand side."
...
+  "The `ivy-height' of `counsel-yank-pop'."
+  :group 'ivy
+  :type 'number)
```
Comment (basil-conto) [inline counsel.el]: This should be `integer` instead of `number`.

File: counsel.el (hunk 1 above)
Comment (jojojames) [reply counsel.el]: Changed.

File: counsel.el (hunk 2 above)
Comment (jojojames) [reply counsel.el]: Thanks, updated.

**Comment by abo-abo** [comment]
Thanks.

**Pull request merged**